2-planning/1-editors.adoc

```diff
@@ -31,7 +31,7 @@ Other editors can take the form of desktop applications, which can be used
 throughout the life-cycle, loading local files and allowing you to edit them at
 commit them back to the repository (or wherever they live) at will.
 
-== OpenAPI Editors
+== Standard HTTP Editors
 
 === Swagger Editor
 
```

Comment:
"hosted" in `Swagger Editor is a very simple hosted text-based editor from` below confused me for a second - the editor can also be used locally no? I've used https://github.com/swagger-api/swagger-editor in the past. Sorry if I'm misunderstanding 😇.

Reply:
Can't edit the blob below. I'd rewrite 
```It can be nice to write specifications on the left and see their rendering as
documentation on the right, but it would possibly be more useful if the right
contained forms for managing the specification beyond modifying text, but the
right panel is purely read-only. ```

as

```
It can be nice to write specifications on the left and see their rendering as
documentation on the right. The right panel is purely read-only and it would possibly be more useful if it contained forms for managing the specification beyond modifying text. 
```

Reply:
(yes I am just catching up with this chapter :P )

Reply:
```
- You are getting validation errors from some local tool which seem weird, so you bundle the multiple files into one file and paste it up here to see what it says
```

Not necessarily the case if you use the Swagger Editor locally (I mentioned this above). But meh, maybe some people don't and they do this dance :D 🤷‍♀️